GOVERNANCE.md

```diff
@@ -61,6 +61,14 @@ For the current list of Collaborators, see the project
 A guide for Collaborators is maintained in
 [COLLABORATOR_GUIDE.md](./COLLABORATOR_GUIDE.md).
 
+Typical activities of a Collaborator include:
+
+* contributing code and documentation changes to improve the project
+* merging pull requests
+* commenting on issues and pull requests
```

Comment:
Possibly, although `reviewing` doesn't necessarily entail leaving an artifact of the review, whereas `commenting` does. I'd be fine with either word.